tests/conftest.py

```
@@ -22,27 +24,62 @@ def data_dir() -> pathlib.Path:
-    logger.info("Downloading the test dataset...")
-    manager = MedImageTestData()
-    latest_release: GitHubRelease = manager.get_latest_release()
-    # banned = ["4D-Lungs", "CC-Tumor-Heterogeneisty.tar"]
-    banned = ["4D-Lung", "CC-Tumor-Heterogeneity.tar"]
-    selected_tests = list(
-        filter(
-            lambda asset: not any(
-                asset.name.startswith(nope) for nope in banned
-            ),
-            latest_release.assets,
+def download_all_test_data(data_dir: pathlib.Path, worker_id) -> dict[str, Path]:
```

Ruff (0.8.2)

Missing type annotation for function argument `worker_id`

(ANN001)